spec/at-rules/extend.md

```diff
@@ -212,15 +218,17 @@ that includes CSS for *all* modules transitively used or forwarded by
     statements in `domestic`'s CSS tree that contains only comments and
     `@import` rules *and* that ends with an `@import` rule.
 
-  * Insert a copy of `initial-imports` in `css` after the last `@import` rule, or
-    at the beginning of `css` if it doesn't contain any `@import` rules.
+  * Insert a copy of `initial-imports` in `css` after the longest initial
+    subsequence of comments and `@import` rules in `css`.
+
+    > If there are no comments or `@import` rules in `css`, this initial
+    > subsequence is empty and `initial-imports` is inserted at the beginning of
+    > `css`.
 
   * For each top-level statement `statement` in `domestic`'s CSS tree after
     `initial-imports`:
 
-    * If `statement` is an `@import` rule, insert a copy of `statement` in `css`
-      after the last `@import` rule, or at the beginning of `css` if it doesn't
-      contain any `@import` rules.
+    * If `statement` is a marked comment, ignore it.
```

Comment:
I don't see where `statement` gets marked, I think "them mark is" is missing in the bullet point below 🤔

Reply:
It's marked on line 210, above.

Reply:
That bullet marks a "comment" rather than a `statement`

Reply:
Right—this bullet checks if `statement` is a comment that is marked.

Reply:
🤦🏽‍♂️ for some reason I read this as "If `statement` is marked, ignore it" lol